circle.yml

```diff
@@ -0,0 +1,8 @@
+machine:
+  xcode:
+    version: "9.0"
+
+test:
+  override:
+    - set -o pipefail && xcodebuild -project Cheers.xcodeproj -scheme "Cheers-iOS" -sdk iphonesimulator clean build
+    - set -o pipefail && xcodebuild -project Cheers.xcodeproj -scheme "Cheers-iOS" -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 8,OS=11.0' -enableCodeCoverage YES test
```

Comment:
You don't need **both** a build and a test command, since test also includes a build 🙂

Reply:
@JohnSundell 👍